index.js

```diff
@@ -32,3 +32,31 @@ export function useStore(store, opts = {}) {
 
   return store.get()
 }
+
+export function useStoreListener(store, opts = {}) {
```

Comment:
it is worth moving the new hook into a separate file: this way it will not affect those who do not need it.